src/OrderFormAcceptanceTests.Actions/Utils/ElementExtensions.cs

```diff
@@ -36,10 +36,13 @@ public static bool ContainsElement(this IWebElement element, By message)
                 driver.FindElements(elementBy)[index], value);
         }
 
+#pragma warning disable IDE0060
         public static void WaitForJsToComplete(this IWebDriver driver, WebDriverWait wait)
         {
             wait.Until(driver => ((IJavaScriptExecutor)driver).ExecuteScript("return document.readyState").Equals("complete"));
         }
+#pragma warning restore IDE0060
+
```

Comment:
Would it make sense to rewrite this as a `WebDriverWait` extension method, so the signature would become something like:

```c#
public static void ForJsToComplete(this WebDriverWait driverWait)
```
and you could call it like `Wait.ForJsToComplete()`.

This would avoid the unused parameter. Alternatively you could make it clear that the `driver` parameter is to be ignored by adding the line `_ = driver;`. However, I think the first suggestion is the better one.